client/common/sass/components/_blog-card-special.sass

```diff
@@ -0,0 +1,57 @@
+.special-blog-card
+	display: grid
+	grid-template: "container"
+	overflow: hidden
+	border-top: 1px solid
+	border-color: $black
+	height: 600px
+	cursor: pointer
+
+	--scale: 1
+	--title-text-decoration: none
+
+	&:hover
+		--scale: 1.08
+		--title-text-decoration: underline
+
+	> *
+		grid-area: container
+
+	img
+		width: 100%
+		height: 100%
+		object-fit: cover
+		place-self: center
+		transition: transform 0.15s ease-in-out
+		transform: scale(var(--scale))
```

Comment:
Thanks for coming up with the CSS for this! Again, it's not super obvious from the Figma that this is there (or maybe it can be seen by drilling down into the Figma element information? Not sure), as visually I wasn't sure if they just carefully chose an image that was darker on the bottom or if a gradient like this was applied.

Reply:
Yes. Usually, you need to select the elements in the Figma design and they have the information about the gradients and other stuff. I do think that editors will need to still chose the image carefully.